docs/source/using.rst

```diff
@@ -2,11 +2,12 @@ Using PyMFE
 ###########
 Extracting metafeatures with PyMFE is easy.                                     
   
-The parameters are the measures, the group of measures and the summarization
-functions to be extracted. The default behavior is to extract all default
-measures, which is. The ``fit`` function can be called by passing the ``X``
-and ``y``. The ``extract`` function is used to extract the related measures.
-See this example::
+The simplest way to extract meta-features is by instantiating the `MFE` class.
+The parameters are the measures, the group of measures, and the summarization
+functions to be extracted. The default parameter is to extract all the
```

Comment:
I think we should keep the "default" word here, and explicitly write which groups are the "default" groups, since the MFE does only extract five meta-feature groups with the default parameters, and not all the groups.

Reply:
done